selfdrive/test/process_replay/test_processes.py

```diff
@@ -22,6 +22,7 @@
   ("GM", "ae3ed0eb20960a20|2021-01-15--15-04-06--8"),          # GM.VOLT
   ("NISSAN", "e4d79cf6b8b19a0d|2021-01-17--14-48-08--7"),      # NISSAN.XTRAIL
   ("VOLKSWAGEN", "18105fc8427a07e1|2021-01-18--21-24-08--12"), # VW.GOLF
+  ("VOLKSWAGEN", "07667b885add75fd|2021-01-23--09-53-10"),     # VW.A3
```

Comment:
remove this, currently we only test one car per make

Reply:
done